Description: Enable Python 3.7 in setup.py
Author: Sophie Brun <sophie@freexian.com>

--- a/setup.py
+++ b/setup.py
@@ -259,6 +259,7 @@ setup(
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'Topic :: Database',
         'Topic :: Software Development',
         'Topic :: Software Development :: Code Generators',
